website/docs/secrets/bootstraping-secrets.mdx

```diff
@@ -5,17 +5,274 @@ hide_title: true
+credentials that are either created manually or available through infrastructure a common scenario
+is a secrets store.
+
+Weave gitops allows you to provision the secret management infrastructure as part of [its capabilities](./setup-secrets-operator.mdx).
+However, in order to provision the secret infrastructure, as any other application that has secrets, we
+need to make the secret it needs to install it.
+
+This type of initial secret is what we call it bootstrapping secret and this guide gives you
+with the different alternatives that weave gitops gives you to achieve the task.
+
+Bootstrapping secrets will be not only present while provision your secrets managmenet solution but in
+any platform provisioning task that comes before you have avaialble before the secrets management solution
+is availabe. Another common example is provisioning platform capabilities itself via
+https://docs.gitops.weave.works/docs/cluster-management/getting-started/#profiles-and-clusters
+that are stored in private repositories. For example https://fluxcd.io/flux/guides/helmreleases/#helm-repository-authentication-with-credentials
+
+To address this problem, weave gitops provides the following alternatives:
+
+- SecretSync CRD: to leverage weave gitops for secrets bootstrapping.
+- SecretSync Terraform Module: to leverage terraform and t for secrets bootstrapping.
 
 ## Using `SecretSync` Custom Resource
-// Description on how SecretSync works and what it provides and its limitations, and maybe an example CR yaml
+
+`SecretSync` is a [Kubernetes Customer Resource](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
+that provides semantics to sync [Kuberentes Secrets](https://kubernetes.io/docs/concepts/configuration/secret/) from management cluster to leaf clusters.
+
+An example could be seen below:
+
+```yaml
+apiVersion: capi.weave.works/v1alpha1
+kind: SecretSync
+metadata:
+  name: my-dev-secret-syncer
+  namespace: default
+spec:
+  clusterSelector:
+    matchLabels:
+      environment: dev
+  secretRef:
+    name: my-dev-secret
+  targetNamespace: my-namespace
+```
+Where you could find two main configuration sections:
+
+1) Select the secret to sync:
+
+```yaml
+  secretRef:
+    name: my-dev-secret
+```
+
+2) Select the [GitopsClusters](https://docs.gitops.weave.works/docs/cluster-management/managing-existing-clusters/)
+,that the secret will be synced to, via labels:
+
+```yaml
+  clusterSelector:
+    matchLabels:
+      environment: dev
+```
+
+More info about the CRD spec [here](./spec/v1alpha1/secretSync.mdx)
+
 ### Working with SecretSync
-// Step by step guide to how to set it up to acheive bootstrapping a secret from management cluster to a new cluster called 'dev'
 
+#### Before start
+
+1. You are using [Weave Gitops Enterprise version > XX](https://docs.gitops.weave.works/docs/releases/) with support for SecretsSync.
```

Comment:
to review with @ahsayde minimum version available

Reply:
ticket to release it https://github.com/weaveworks/weave-gitops-enterprise/issues/2500

Reply:
released and tested